stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -377,12 +401,13 @@ function externEncryptRsaEcb(byte[] input, PrivateKey|PublicKey key, handle padd
 } external;
 
 # Returns AES CBC encrypted value for the given data.
+# ```ballerina byte[]|crypto:Error cipherText = crypto:encryptAesCbc(cipherText, key, initVector); ```
 #
 # + input - The content to be encrypted
 # + key - Encryption key
 # + iv - Initialization vector
 # + padding - The padding
-# + return - Encrypted data or `Error` if key is invalid
+# + return - Encrypted data or `crypto:Error` if key is invalid
```

Comment:
```suggestion
# + return - Encrypted data or `crypto:Error` if key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.